src/Uno.UI.RuntimeTests/IntegrationTests/common/TestServices.WindowHelper.cs

```diff
@@ -8,6 +8,7 @@
 using Windows.UI.Xaml.Tests.Enterprise;
 using Windows.UI.Core;
 using MUXControlsTestApp.Utilities;
+using Uno.UI.Extensions;
```

Comment:
The addition of this using caused:

```
/Users/runner/work/1/s/src/Uno.UI.RuntimeTests/IntegrationTests/common/TestServices.WindowHelper.cs(167,49): error CS0121: The call is ambiguous between the following methods or properties: 'Uno.UI.Extensions.DependencyObjectExtensions.FindFirstChild<T>(Windows.UI.Xaml.DependencyObject, int?, bool)' and 'Uno.UI.ViewExtensions.FindFirstChild<T>(Android.Views.ViewGroup, int?, bool)' [/Users/runner/work/1/s/src/Uno.UI.RuntimeTests/Uno.UI.RuntimeTests.netcoremobile.csproj::TargetFramework=net6.0-android]
```

Reply:
@dr1rrb I pushed a commit to workaround that

Reply:
Thanks @Youssef1313 👍